### PR TITLE
Abb20

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record for Abb 20 and minor corrections. Abbadie 20 is a bit complex due to the scribal hands involved. 
Carsten, I placed identifications in a note, unless there are IDs, and have removed "space holders". Maybe there are a few pieces that can be added to the bibliography.

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

File: ParisBNF/abb/BNFabb20.xml [hunk 1]
```diff
@@ -52,60 +93,212 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                                    <note>The leaves <locus target="#1 #2 #128"/> are not ruled.</note>
+                                </layout>
+                            </layoutDesc>
                         </objectDesc>
+                        <handDesc>
+                            
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <locus from="3ra" to="125vb"/>
+                                <locus from="126rb_l19" to="128rc"/>
+                                <seg type="script">End of the 18th/beginning of the 18th century according to C. Conti Rossini, and 18th/19th century according to
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb20.xml:128]: "End of the 18th/beginning of the **19th** century according to C. Conti Rossini,"

File: ParisBNF/abb/BNFabb20.xml [hunk 2]
```diff
@@ -52,60 +93,212 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                                                    Chaldäer, Sini (= Chinesen), Ḫāmāzi (= Himyaren) und Araber. An Gewässern
-                                                    haben sie den Euphrat und den Strom Bäläkä (= Balch).</ab>
+                                                    die Syrer, die Gebiete von Rabiʿa, die Insel Waḥarāmqā, die Chaldäer - das sind die Leute von Babylon, Persien, Khorasan,
+                                                    Feryā, Sind, Indien, Jemen, Ṭayǝf, Tamāmā sowie zwei Meere und die Araber. Diejenigen, die mit Buchstaben
+                                                    zu schreiben wissen, sind acht: Hebräer, Śǝyanāwi, Perser, Inder,
+                                                    Chaldäer, Sini, Ḫāmāzi und Araber. An Gewässern
+                                                    haben sie den Euphrat und den Strom Balaka.</ab>
                                     </q>
+                                    <note>Rabiʿa should be understood as Diyar Rabia, the island Waḥarāmqā as Ǧarāmiqa, Feryā as Fergana, Ṭayf as aṭ-Ṭayn, Tamāmā as
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb20.xml:177]: Delete: "Ṭayf as aṭ-Ṭayn". 
This identification is outdated. Ṭayf is more likely Ṭayfa, the Arabic oasis and ancient trading centre.

**Review by CarstenHoffmannMarburg — COMMENTED** [review]
Thank you for improving this description!

File: ParisBNF/abb/BNFabb20.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb20.xml:128]: ok

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**